backend/posts/bc.py

```diff
@@ -79,9 +77,10 @@ def get_comment(self, index):
 
     def verify_comment(self, comment: Comment, post: Post):
         comments_count = self.comments_count()
+        comment = CommentStoreContract.Comment(comment.data_hash, comment.post_hash)
         for comment_index in reversed(range(comments_count)):
             stored_comment = CommentStoreContract.Comment(*self.get_comment(comment_index))
-            if stored_comment.data_hash == comment.data_hash and stored_comment.post_hash == post.data_hash:
+            if stored_comment == comment:
                 comment.verified = True
                 comment.save()
                 break
```

Comment:
In Python files you should not never leave no newline at the end (no trailing line).